packages/manager/apps/web/client/app/hosting/ftp/FTP.html

```diff
@@ -357,7 +364,29 @@
                         <dt
                             data-translate="hosting_tab_FTP_dashboard_login"
                         ></dt>
+                        <div
+                            data-ng-if="ctrlHostingTabFTP.primaryUser.ftp.user !== ctrlHostingTabFTP.primaryUser.ssh.user"
+                            class="mb-3"
+                        >
+                            <div>
+                                <span
+                                    data-translate="hosting_tab_FTP_FTP_user"
+                                ></span>
+                                <span
+                                    data-ng-bind="ctrlHostingTabFTP.primaryUser.ftp.user"
+                                ></span>
+                            </div>
+                            <div>
+                                <span
+                                    data-translate="hosting_tab_FTP_SSH_user"
+                                ></span>
+                                <span
+                                    data-ng-bind="ctrlHostingTabFTP.primaryUser.ssh.user"
+                                ></span>
+                            </div>
+                        </div>
```

Comment:
**nitpick:** Include it as a part of `dt` or `dd` whichever is appropriate.

Reply:
Hello, thanks for the review, I updated div to dd